tritium-core/src/main/java/com/palantir/tritium/event/CompositeInvocationEventHandler.java

```diff
@@ -32,6 +32,36 @@
 
     private static final Logger logger = LoggerFactory.getLogger(CompositeInvocationEventHandler.class);
 
+    // A sentinel value is used to differentiate null contexts returned by handlers from
+    // invocations on disabled handlers.
+    private static final InvocationContext DISABLED_HANDLER_SENTINEL = new InvocationContext() {
```

Comment:
nit: should this be a singleton enum shared with ByteBuddy instrumentation?

Reply:
It can be if we're okay with making it public. It's an implementation detail which isn't necessary in every case, so I'd prefer if it wasn't possible to reference externally.

Reply:
+1 for singleton enum